pandas/plotting/_core.py

```diff
@@ -721,6 +721,10 @@ def __call__(self, *args, **kwargs):
             plot_backend.__name__, self._parent, args, kwargs
         )
 
+        # when using another backend, get out of the way
+        if plot_backend.__name__ != "pandas.plotting._matplotlib":
+            return plot_backend.plot(self._parent, x=x, y=y, kind=kind, **kwargs)
+
         kind = self._kind_aliases.get(kind, kind)
```

Comment:
This should probably be done before dispatching.

Reply:
I deferred it so that hvplot can have more kinds than matplotlib. Is that fair?

Reply:
Ahh... so maybe this exact line should be done, so that things like `kind='density'` and `kind='kde'` both work.

But the

```
        if kind not in self._all_kinds:
            raise ValueError("{} is not a valid plot kind".format(kind))

```

should *not* be done before dispatching, so that additional kinds are allowed.

Reply:
oh yeah. That makes sense.